public/app/plugins/datasource/prometheus/querybuilder/components/promQail/state/helpers.ts

```diff
@@ -19,7 +19,7 @@ import { Interaction, QuerySuggestion, SuggestionType } from '../types';
 import { createInteraction, stateSlice } from './state';
 
 const OPENAI_MODEL_NAME = 'gpt-3.5-turbo';
-const promQLTemplatesCollection = 'promql:templates';
+const promQLTemplatesCollection = 'grafana:core:promql_templates';
```

Comment:
Hmm don't really like this as `grafana:core` is reserved for grafana (internal) metadata
Let's fix the table in dev db instead of changing it here

Reply:
I've renamed to `grafana.promql.templates` as we decided on Slack